common/util.go

```diff
@@ -606,18 +607,32 @@ func CheckEventBlobSizeLimit(
 
 	scope.RecordTimer(metrics.EventBlobSize, time.Duration(actualSize))
 
-	if actualSize > warnLimit {
+	if errorLimit < warnLimit {
 		if logger != nil {
```

Comment:
is there a reason we wouldn't have a logger?  seems kinda unlikely / something we should just fix...

Reply:
Good point. I'm not sure why this existed before. I'll remove it.